glslang/MachineIndependent/SpirvIntrinsics.cpp

```diff
@@ -42,7 +41,6 @@
 #include "../Include/SpirvIntrinsics.h"
 #include "../Include/Types.h"
 #include "ParseHelper.h"
-
```

Comment:
nit: probably shouldn't remove this newline

Reply:
Solved